previewers/betatest/js/video.js

```diff
@@ -9,5 +9,106 @@ function translateBaseHtmlPage() {
 
 function writeContent(fileUrl, file, title, authors) {
     addStandardPreviewHeader(file, title, authors);
-    $(".preview").append($("<video/>").prop("controls",true).append($('<source/>').attr("src",fileUrl)));
+
+    const queryParams = new URLSearchParams(window.location.search.substring(1));
```

Comment:
This only works when signedUrls aren't used. Otherwise, the id and siteUrl aren't sent as params and those variables get null below, leading to error messages when the ajax call at line 25 is made. Can you add an if siteUrl is null check that would also just revert to the fallback in line 35 to avoid that error message?